admin/menu.php

```diff
@@ -42,6 +42,29 @@ function extensions() {
 }
 
 /**
+ * Highlights the 'Backups' submenu item when on the Extentions page
+ *
+ * @param string $parent_file The filename of the parent menu item
+ * @return string $parent_file The filename of the parent menu item
+ */
+function highlight_submenu( $parent_file ) {
+
+	global $submenu_file;
+
+	$screen = get_current_screen();
+
+	if ( 'tools_page_' . HMBKP_PLUGIN_SLUG . '_extensions' === $screen->id ) {
+
+		// Set the main plugin page to be the active submenu page
+		$submenu_file = HMBKP_PLUGIN_SLUG;
+
+	}
+
+	return $parent_file;
+}
+add_filter( 'parent_file', 'HM\BackUpWordPress\highlight_submenu' );
```

Comment:
@sambulance Would it be more appropriate to use the `submenu_file` filter instead ? Since you're not using the `parent_file` variable anyway. Would drop the need for the global var as well.